UMLEditor/UMLEditor/Views/MainWindow.axaml.cs

```diff
@@ -977,31 +976,29 @@ private void Add_Method_OnCLick(object sender, RoutedEventArgs e)
                     // Taking the className string and accessing a corresponding class in active diagram
                     Class currentClass = _activeDiagram.GetClassByName(className);
                     
-                    // If the class does not exist...
-                    if (currentClass is null)
```

Comment:
(Referring to line 941)
- messageTitle should be windowTitle in the call to CreateDialog (bug in RaiseAlert)
- The A in AlertDialog should be lowercased